Modules/DatabricksPS/Public/SecretsAPI.ps1

```diff
@@ -46,8 +46,10 @@ Function Add-DatabricksSecretScope {
     $parameters = @{
       scope = $ScopeName 
     }
-	
-    $parameters | Add-Property -Name "initial_manage_principal" -Value $InitialManagePrincipal
+    
+    if ($InitialManagePrincipal) {
```

Comment:
this change is not necessary as `Add-Property` omits NULL/empty values automatically

Reply:
Just removed the if condition as requested.